docs/upgrade/_common/upgrade-image-enterprise.rst

```diff
@@ -0,0 +1,36 @@
+There are two alternative upgrade procedures:
+
+* :ref:`Upgrading ScyllaDB and simultaneously updating 3rd party and OS packages <upgrade-image-recommended-procedure>`. It is recommended if you are running a ScyllaDB official image (EC2 AMI, GCP, and Azure images), which is based on Ubuntu 20.04.
+
+* :ref:`Upgrading ScyllaDB without updating any external packages <upgrade-image-enterprise-upgrade-guide-regular-procedure>`.
+
+.. _upgrade-image-recommended-procedure:
+
+**To upgrade ScyllaDB and update 3rd party and OS packages (RECOMMENDED):**
+
+.. versionadded:: 2021.1.10
+
+Choosing this upgrade procedure allows you to upgrade your ScyllaDB version and update the 3rd party and OS packages using one command. 
+
+#. Update the |SCYLLA_REPO|_ to |NEW_VERSION|.
+
```

Comment:
@annastuchlik  After the following line `Update the ScyllaDB Enterprise Repo to 2022.1` we need to add additional step `sudo apt-get update` to load the new repo. otherwise the next command will fail to find the packages